mpfr/PKGBUILD

```diff
@@ -9,6 +9,7 @@ if [ -n "${_patchlevel}" ]; then
 fi
 #pkgver=${_pkgver}
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `mpfr` Therefore this change should be unnecessary.